src/common/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import CloseIcon from "@/images/ic_close.svg";
+import styles from "./badge.module.scss";
+import classNames from "classnames";
+
+interface BadgeProps extends React.HTMLAttributes<HTMLDivElement> {
+  color?: "blue" | "green" | "red";
+  mobile?: boolean;
```

Comment:
size라던지 다른 네이밍으로 표현되면 더 좋을 것 같습니다!
```suggestion
  mobile?: boolean;
```